.changeset/spicy-parrots-raise.md

```diff
@@ -0,0 +1,7 @@
+---
+'@solana/transaction-messages': patch
+'@solana/instructions': patch
+'@solana/errors': patch
```

Comment:
Patch is debatable. I'm widening the type of `IInstruction`, so any code that creates one of those is unaffected, but any code that treats it as mutable is not. In our code I needed to update some type params. Downstream consumers like Codama clients need to be updated to avoid any code changes for their dependencies. Happy to change this to whatever we think is best.